cli_tools/common/utils/daisy/daisy_utils_test.go

```diff
@@ -151,6 +153,28 @@ func testRemovePrivacyTag(t *testing.T, originalMessage string, expectedMessage
 	}
 }
 
+func TestUpdateToUEFICompatible(t *testing.T) {
+	w := createWorkflowWithCreateDiskImageAndIncludeWorkflow()
+
+	UpdateToUEFICompatible(w)
+
```

Comment:
I think it'd be clearer to put the whole structs into this method: one for before update, and one for what you expect after the update, and then do a single assertion that the result is what you expected. The alternative below makes it difficult for me to track where the differences are expected to be.

Reply:
What I'm trying to say is that to read the assertions in this test I have to do a lot of parsing and array indexing in my head and I think that having the two structs printed out in front of me would be easier. But this isn't the highest priority thing, and if you like this, then I'm fine.

Reply:
There are no arrays to parse in the assert, only 3 arguments: t, expected, actual.

Reply:
Parsing in my head